pacman.sh

```diff
@@ -14,10 +14,10 @@ set -o pipefail
 
 if [ -x /usr/bin/checkupdates ]
 then
-    updates=$(/usr/bin/checkupdates | wc -l)
+    updates=$( (checkupdates || true) | wc -l)
```

Comment:
`|| true` ignores all failures. As-is it would even pick another `checkupdates` executable if found in `$PATH`. Isn't there a specific exit code? If need be the output can also be checked.